src/linear_algebra/finite_dimensional.lean

```diff
@@ -125,6 +125,10 @@ begin
   exact s_basis,
 end
 
+/-- A finite dimensional vector space over a finite field is finite -/
+noncomputable def finite_of_findim_over_finite [fintype K] [hE : finite_dimensional K V] : fintype V :=
```

Comment:
Since this is in the namespace `finite_dimensional`, we can just copy the name for `module`.
```suggestion
noncomputable def fintype_of_fintype [fintype K] [finite_dimensional K V] : fintype V :=
```

Reply:
Fixed.